app/experimenter/experiments/tests/factories.py

```diff
@@ -11,6 +11,7 @@
 
 
 class ExperimentFactory(factory.django.DjangoModelFactory):
+    created_date = factory.LazyAttribute(lambda o: datetime.datetime.now())
```

Comment:
Actually you don't need to do this at all, the model will auto populate it, that's what auto_now_add does.